src/checkHoliday.js

```diff
@@ -28,30 +31,30 @@ nthWeekdayOfMonth = (weekday, n, date) => {
 
 writeHolidays = () => {
     nationalHolidays.forEach(holiday => {
-        let holidayMonth = new Date(holiday.date).getMonth();
+        const holidayMonth = new Date(holiday.date).getMonth();
         if (holiday.type === 'public' && holidayMonth !== 0 && holidayMonth !== 11 && holiday.name != 'Remembrance Day' && holiday.name != 'Easter Sunday') {
             //console.log(holiday);
-            employeeHolidays.push(holiday.date.slice(0,10));
+			holidayJSON["holidays"].push(holiday.date.slice(0,10));
         }
     });
 
     // Add family day to the list of holidays
-    const familyDayDate = nthWeekdayOfMonth(1, 3, new Date(new Date().getFullYear(), 1)).toISOString().slice(0,10);
-    employeeHolidays.push(familyDayDate);
-    
-    const currentYear = new Date(new Date().getFullYear());
-    const firstWinterHolidayDate = nthWeekdayOfMonth(1, 4, currentYear, 11);
-    const lastWinterHolidayDateThisYear = nthWeekdayOfMonth(5, 1, currentYear, 0);
-    const lastWinterHolidayDateNextYear = nthWeekdayOfMonth(5, 1, currentYear + 1, 0);
+    const familyDayDate = nthWeekdayOfMonth(1, 3, new Date(currentYear, 1)).toISOString().slice(0,10);
```

Comment:
Consider creating an `Intl` formatter instead of having to use `toISOString` and having to `slice`. 

E.g. 
```
const formatter = new Intl.DateTimeFormat('en-CA', { literal: '-' });
formatter.format(new Date())   // 2019-12-20
``` 
can be reused anywhere you need to format a date.